pkg/scheduler/eventhandlers_test.go

```diff
@@ -26,7 +26,14 @@ import (
 	"k8s.io/api/core/v1"
 	"k8s.io/apimachinery/pkg/api/resource"
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apiserver/pkg/util/feature"
+	testing2 "k8s.io/component-base/featuregate/testing"
```

Comment:
`featuregatetesting "k8s.io/component-base/featuregate/testing"` is more conventional.

Reply:
done